zstacklib/zstacklib/utils/qcow2_luks.py

```
@@ -0,0 +1,109 @@
+
+
+def find_volume_secret_uuid(volume_path):
+    for suuid in list_secret_uuids():
+        try:
+            xml = shell.call("virsh secret-dumpxml %s" % suuid)
+            if re.search(
+                    r"<usage\s+type=['\"]volume['\"]>.*?<volume>%s</volume>" % re.escape(volume_path),
+                    xml, re.S):
+                return suuid
+        except Exception:
+            continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)